src/Filter/ModelFilter.php

```diff
@@ -15,6 +15,9 @@
 
 use Doctrine\Bundle\MongoDBBundle\Form\Type\DocumentType;
 use Doctrine\Common\Collections\Collection;
+use Doctrine\ODM\MongoDB\Mapping\ClassMetadata;
+use MongoDB\BSON\ObjectId;
```

Comment:
Looks unused in this scope, you are using \ObjectId

Reply:
I used \ObjectId in PHPDoc, but it was a mistake, this is the class: https://www.php.net/manual/en/class.mongodb-bson-objectid.php

Reply:
Could you the fix the PHPDoc please, and also check the other files? Thanks

Reply:
sure! I realised of that when you told me and it is already fixed